django/forms/boundfield.py

```diff
@@ -105,6 +105,8 @@ def as_widget(self, widget=None, attrs=None, only_initial=False):
             )
         else:
             value = self.value()
+        if not widget.is_hidden and self.errors:
+            attrs["aria-invalid"] = "true"
```

Comment:
We should probably move this to the `build_widget_attrs()`, check out 3f73df44f2726319f6af44c5613e5e531e9370b6.

Reply:
Yes. That makes sense. 👍